vividus-plugin-web-app/src/main/java/org/vividus/selenium/BrowserWindowSize.java

```diff
@@ -61,6 +47,6 @@ public Dimension toDimension()
     @Override
     public String toString()
     {
-        return sizeAsString;
+        return getWidth() + "x" + getHeight();
```

Comment:
You mean we need separate unit tests for BrowserWindowSize ? 
Because looks like line is covered implicitly.
https://codecov.io/gh/vividus-framework/vividus/compare/de6b6ecf149cc7833aa115cda423036992f97391...cb08258fabc833145dd5e14c43c5536a91729931/diff

Reply:
Sorry, missed that case